python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -114,15 +114,21 @@ def _attribute_filter__set__(self, name, value):
             #   Do some checking of the filenames in a subprocess
             if name == 'lfn':
                 self.namePattern = os.path.basename(value)
-                self.remoteDir = os.path.dirname(value)
+                if os.path.dirname(value):
+                    self.remoteDir = os.path.dirname(value)
+                elif os.path.isfile(os.path.join(os.getcwd(), self.namePattern)):
+                    self.remoteDir = os.getcwd()
```

Comment:
Auto-discovery of files if they exist in the cwd as proven quite useful to users. I can kill the feature but I'm opting to keep it in.